package.json

```diff
@@ -1,43 +1,44 @@
 {
-  "name": "nmea0183-signalk",
-  "version": "0.2.4",
+  "name": "signalk-parser-nmea0183",
+  "version": "2.0.6",
```

Comment:
I started counting from 2 and up, and this is the 6th minor release (in terms of features) of the 2.0.x branch.